src/components/Button/index.tsx

```diff
@@ -1,17 +1,16 @@
-import { ButtonHTMLAttributes } from 'react';
+import React, { ButtonHTMLAttributes } from 'react';
+
 import { StyledButton } from './styles';
 
-type Variants = 
-|'filled'
-|'ghost'
+type Variants = 'filled' | 'ghost';
 
 type ButtonProps = Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'css'> & {
```

Comment:
*interface

Reply:
Esse caso é bem específico, mas talvez funcione com `Interface ButtomProps extends Omit<Buttton...>`, faz o teste, caso não role, deixa assim mesmo.

Reply:
omit only refers to a type =/

Reply:
Tá, então deixa assim...